src/genesis.rs

```diff
@@ -0,0 +1,43 @@
+use crate::cli::Network;
+use blockfrost_openapi::models::genesis_content::GenesisContent;
+
+pub fn get_genesis_content_for(network: &Network) -> GenesisContent {
```

Comment:
Definitely a wanna change for the future.
It would be nice to be able to use custom networks here, by reading the genesis from a file?
While working in the Lace team, I had problem integrating blockfrost-ryo into our CI and local test runs due to hard coded network values. I had to modify the source code.

Imagining a similar case for blockfrost-platform, we would want to change this in the future.